packages/plugins/graphql/server/services/utils/attributes.js

```diff
@@ -9,10 +9,7 @@ module.exports = ({ strapi }) => {
    * @return {boolean}
    */
   const isStrapiScalar = attribute => {
-    return strapi
-      .plugin('graphql')
-      .service('constants')
-      .STRAPI_SCALARS.includes(attribute.type);
+    return strapi.plugin('graphql').service('constants').STRAPI_SCALARS.includes(attribute.type);
```

Comment:
Looks like the formatting broke, could you revert those style changes to match our prettier config, please? (same for the other kind of changes)